src/Education.UnitTests/XamlGenerator/XamlGeneratorHelperTests.cs

```diff
@@ -132,7 +132,7 @@ public void WriteDocumentHeaderAndEndDocument_ProduceCorrectStructure_NewCCT()
         </TextBlock>
       </Border>
       <TextBlock>
-        <Hyperlink NavigateUri=""https://docs.sonarsource.com/sonarlint/visual-studio/concepts/clean-code"">Learn more about Clean Code</Hyperlink>
```

Comment:
How was this lost? Are there any other things that we need to check?

Reply:
due to comment/uncomment of the test code. There was a conflict here and I missed the URL change.